.php-cs-fixer.php

```diff
@@ -11,12 +21,13 @@
 $finder = PhpCsFixer\Finder::create()
```

Comment:
It seems to me that the `Finder::create` is also deprecated

Reply:
I could not find infos about that. 
PhpCsFixer extends the symfony finder.
https://github.com/symfony/finder/blob/6.0/Finder.php#L76

Can you please give some hints about the deprecation?

Reply:
Not found either, bad memory sorry 😅